<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1398</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #d7dfe0;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="set_image_database_path_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #329ba8 };</string>
    </property>
    <property name="text">
     <string>set image database path</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="search_folder_recursively_checkbox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>search folder recursively</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_db_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Path: _</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SUB IMAGE WIDTH:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SUB IMAGE HEIGHT:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sub_image_width_input">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white };</string>
    </property>
    <property name="text">
     <string>281</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sub_image_heigt_input">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white };</string>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>width and height every images will be converted to. The image will be scaled down later on according to the dimensions of the main image</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAGES PER ROW:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="images_per_row">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { background-color: white };</string>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="do_not_reuse_subimages_checkbox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>do not reuse subimages</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_image_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #329ba8 };</string>
    </property>
    <property name="text">
     <string>Save image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_and_index_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #329ba8 };</string>
    </property>
    <property name="text">
     <string>Convert and index images</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_main_image_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #329ba8 };</string>
    </property>
    <property name="text">
     <string>Load main image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_random_converted_image_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #329ba8 };</string>
    </property>
    <property name="text">
     <string>Show random converted image</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>1161</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
	background-color: white;
	border: 1px solid black;
 };</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>1371</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="feedback">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>1371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit { 
	background-color: white;
	border: 1px solid black;
 };</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>-10</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount of images user per row for the main image. The amount per column will be calculated accordingly</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount of images:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="artify_main_image_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #329ba8 };</string>
    </property>
    <property name="text">
     <string>Artify main image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1398</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
